src/components/ConversationSettings/ListableSettings.vue

```diff
@@ -85,12 +92,28 @@ export default {
 		conversation() {
 			return this.$store.getters.conversation(this.token) || this.$store.getters.dummyConversation
 		},
+
+		summaryLabel() {
+			switch (this.listable) {
+			case CONVERSATION.LISTABLE.ALL:
+				return t('spreed', 'Conversation is open to registered and guest app users')
+			case CONVERSATION.LISTABLE.USERS:
+				return t('spreed', 'Conversation is open to registered users')
+			case CONVERSATION.LISTABLE.NONE:
+			default:
+				return t('spreed', 'Conversation is limited to the current participants')
+			}
+		}
 	},
 
 	watch: {
 		value(value) {
 			this.listable = value
 		},
+
+		conversation() {
+			this.listable = this.conversation.listable
+		},
```

Comment:
```suggestion
		conversation: {
			immediate: true,
			handler() {
				this.listable = this.conversation.listable
			},
		},
```

It will stay as `null`

Reply:
And btw , I tested it on an open conversation that is accessible to guests as well. However, the `listable` attribute is set as 1 ( CONVERSATION. LISTABLE.USERS) for conversation object. 
Please check how this value is set for conversation perhaps there is a bug.
![image](https://github.com/nextcloud/spreed/assets/84044328/9a00d5ee-7472-4243-a479-fd8f8605f3cb)

Reply:
Aaa never mind, I understand the difference now between Guests app users and guests